lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -82,6 +128,75 @@ def self.serialize(store:, name:, serializer: nil)
         store_accessor store, name
       end
 
+      def buffers
+        pm_storage_adapter.buffers
+      end
+
+      def self.create_later(attrs, storage_adapter)
+        element = new(attrs)
+        storage_adapter.buffers[:upsert][element.unique_identifier] = element
+
+        # TODO: Why did we have to do this again? Don't get it
```

Comment:
Discussed this offline, this goes into part of a more general TODO to support non-upsertable databases like Postgres 9.4